README.md

```diff
@@ -13,13 +13,13 @@ service, please see the following links:
 
 ## Quickstarts
 
-* If you have a whitelisted subscription and want to use `az aro` to create a
-  cluster using the production RP, follow [using `az
-  aro`](docs/using-az-aro.md).
+* If you have a whitelisted subscription and want to create a 4.3 cluster using
+  the production RP, follow [Create, access, and manage an Azure Red Hat OpenShift 4.3 Cluster][1].
```

Comment:
* If you are an end user and want to create an Azure Red Hat OpenShift 4 cluster, follow [Create, access, and manage an Azure Red Hat OpenShift 4.3 Cluster][1].